packages/video_player/lib/video_player.dart

```diff
@@ -85,9 +86,18 @@ class VideoPlayerValue {
   /// Is null when [initialized] is false.
   final Size size;
 
+  /// The [rotationDegrees] of the currently loaded video.
+  /// Possible values are: 0 or 180 for videos recorded in landscape format,
+  /// 90 and 270 for videos taken in portrait mode.
+  ///
+  /// Is null when [initialized] is false.
+  final int rotationDegrees;
```

Comment:
When will client code look at this?
Should the platform code not just make sure that the video is shown correctly, and then the aspect-ratio can be queried to decide if it is landscape or portrait?
In any case I would call it `rotation`, and document that it is in degrees in the doc-comment

Reply:
`rotationDegrees` is not currently use in dart land and I don't currently have a valid use case for this. I'll remove it from video_player.dart. `rotationDegrees` is only used in native code to switch the `width` and `height` so the aspect ratio is always `width / height`.